contracts/RMRK/extension/erc20Holder/IERC20Holder.sol

```diff
@@ -19,9 +21,9 @@ interface IERC20Holder {
 
     /**
      * @notice This emits when a token transfers ERC-20 tokens.
-     * @param fromTokenId The token that owned the ERC-20 tokens.
-     * @param to The address that receives the ERC-20 tokens.
      * @param erc20Contract The ERC-20 contract.
+     * @param fromTokenId The token that owned the ERC-20 tokens.
```

Comment:
Would you mind removing `.`s from `@param` tags? There are a lot to create suggestions for all of them 😅